core/src/main/java/org/axonframework/eventhandling/MultiEventHandlerInvoker.java

```diff
@@ -80,4 +80,18 @@ public void handle(EventMessage<?> message, Segment segment) throws Exception {
             }
         }
     }
+
+    @Override
+    public boolean supportsReset() {
+        return delegates.stream()
+                        .map(EventHandlerInvoker::supportsReset)
```

Comment:
why not just `delegates.stream().anyMatch(EventHandlerInvoker::supportsReset)`? That would allow it to stop checking after the first "true".

Reply:
Didn't cross my mind, fixed.